go-runtime-builder/builder/go-build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# go-build.sh runs Go build in the workspace.
+# usage: go-build.sh WORKSPACE
+
+set -e
+
+workspace="$1"
+cd "$workspace"
```

Comment:
It's nicer to just be consistent.  Use of ${var} is more flexible as it allows for ${var}something.